src/main/java/org/folio/circulation/domain/anonymization/config/LoanAnonymizationConfigurationForTenant.java

```diff
@@ -0,0 +1,63 @@
+package org.folio.circulation.domain.anonymization.config;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getNestedIntegerProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getNestedStringProperty;
+
+import org.folio.circulation.domain.policy.Period;
+
+import io.vertx.core.json.JsonObject;
+
+public class LoanAnonymizationConfigurationForTenant {
```

Comment:
As every operation in FOLIO is typically performed for a tenant, I think that the `ForTenant` aspect of this configuration can be implied.
```suggestion
public class LoanAnonymizationConfiguration {
```

Reply:
Done